ci/docker/benchmark_build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Builds a docker image benchmark-dev:latest containing the benchmark scripts
+
+# Stop on errors.
+set -e
+
+# NOTE: explicit no -x for verbose commands. Because this is run in CI, doing so may result in
+# publishing sensitive information into public CI logs if someone makes a change in a
+# consuming script that is off guard.
+
+DOCKER_NAME="benchmark"
```

Comment:
I wonder, should we leave a trace of the source in this name? E.g. 'nighthawk-benchmark'?

Reply:
Certainly can update this.